inc/customizer.php

```diff
@@ -1042,12 +1043,18 @@ function estore_customize_partial_blogdescription() {
 	bloginfo( 'description' );
 }
 
-// Function for top eader text selective refresh support
+// Function for top header text selective refresh support
 function estore_bar_text() {
 	$header_bar_text = get_theme_mod( 'estore_bar_text' );
 	echo wp_kses_post( $header_bar_text );
 }
 
+function estore_bar_text_render() {
+	_deprecated_function( __FUNCTION__, '1.4.6', 'estore_bar_text()' );
```

Comment:
This code says that `estore_bar_text_render` is a deprecated function which is not really.

Please read this on how it works.
https://developer.wordpress.org/reference/functions/_deprecated_function/